.github/workflows/ros2.yml

```diff
@@ -68,9 +68,9 @@ jobs:
       uses: ros-tooling/action-cloudwatch-metrics@0.0.4
       with:
         namespace: RobotWorkspaceBuild
-        metric-value: ${{ ! contains(needs.build_and_bundle_ros2.outputs.robot_ws_build_result, 'failure') }}
```

Comment:
I am trying to see cases when `log_workflow_status_to_cloudwatch` gets executed in this workflow. Assuming we restrict push on `ros1` `ros2` branches, this never gets executed? In that case, do we need it?

Reply:
For both `ros1` and `ros2` branches, we're currently logging pass/fail build on `push` events. This `log_workflow_status_to_cloudwatch` is used for logging pass/fail build for `push` events on the `ros2` branch.

Reply:
My question was that are 'push' events possible on `ros2` branch? I was not aware that merging the `pull_request` triggers the `push` event but it does.